images/macos/toolsets/toolset-12.json

```diff
@@ -1,6 +1,6 @@
 {
     "xcode": {
-        "default": "13.3.1",
+        "default": "13.4.1",
```

Comment:
we can't do this without a proper announcement

Reply:
Brought back the old version of xcode